src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js

```diff
@@ -147,7 +147,7 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
                 data: data.fileReader.result.replace(/^.*;base64,/, ''),
             };
 
-            if (data.file.type.startsWith('image/')) {
+            if (data.file.type.startsWith('image/') && !data.file.type.startsWith('image/svg')) {
```

Comment:
so if someone will change the default MFU config to use the image content type for SVG he will not get the alternative text?

Reply:
in that case, it will not have the alt text added which is set as the filename anyway. However, if it is required, it will occur in error like now. So maybe let's edit the information returned from rest whether the indicated type has the ezimage field (and add alternativeText based on it)?

Reply:
Yes, we should have a more reliable way of checking if the alternative text should be added.